deploy/template.yaml

```diff
@@ -209,33 +209,33 @@ Resources:
-              CORE_STUB_URL: "/perfTest/coreStubUrl"
-              KBV_URL: "/perfTest/kbvUrl"
-              ADDRESS_URL: "/perfTest/addressUrl"
-              ADDRESS_ENV_NAME: "/perfTest/addressEnvName"
-              KBV_ENV_NAME: "/perfTest/kbvEnvName"
-              CORE_STUB_USERNAME: "/perfTest/coreStubUsername"
-              CORE_STUB_PASSWORD: "/perfTest/coreStubPassword"
-              CORE_URL: "/perfTest/coreUrl"
-              ORCH_STUB_URL: "/perfTest/orchStubUrl"
-              RP_STUB: "/perfTest/rpStub"
-              BASE_URL: "/perfTest/baseUrl"
-              F2F_URL: "/perfTest/f2fUrl"
-              K6_DYNATRACE_URL: "/perfTest/dynatraceUrl"
-              HOME_URL: "/perfTest/homeUrl"
-              SIGNIN_URL: "/perfTest/signInUrl"
-              EMAIL_OTP: "/perfTest/fixedEmailOTP"
-              PHONE_OTP: "/perfTest/fixedPhoneOTP"
-              CURR_PHONE: "/perfTest/currPhone"
-              NEW_PHONE: "/perfTest/newPhone"
+              MOBILE_TEST_CLIENT_EXECUTE_URL: "/perfTest/mobile/testClientUrl"
+              MOBILE_BACK_END_URL: "/perfTest/mobile/backendUrl"
+              MOBILE_FRONT_END_URL: "/perfTest/mobile/frontendUrl"
+              MOBILE_BIOMETRIC_SESSION_ID: "/perfTest/mobile/biometricSessionID"
+              ACCOUNTS_APP_PASSWORD: "/perfTest/account/testUserPassword"
+              ACCOUNTS_APP_PASSWORD_NEW: "/perfTest/account/testUserNewPassword"
+              ACCOUNTS_EMAIL_OTP: "/perfTest/account/fixedEmailOTP"
+              ACCOUNTS_PHONE_OTP: "/perfTest/account/fixedPhoneOTP"
+              ACCOUNTS_AUTH_APP_KEY: "/perfTest/account/accounts/authAppKey"
+              ACCOUNTS_HOME_URL: "/perfTest/account/accounts/homeUrl"
+              ACCOUNTS_SIGNIN_URL: "/perfTest/account/accounts/signInUrl"
+              ACCOUNTS_CURR_PHONE: "/perfTest/account/accounts/currPhone"
+              ACCOUNTS_NEW_PHONE: "/perfTest/account/accounts/newPhone"
+              ACCOUNTS_RP_STUB: "/perfTest/account/authentication/rpStub"
+              ACCOUNTS_BASE_URL: "/perfTest/account/authentication/baseUrl"
+              IDENTITY_coreStub: "/perfTest/identity/coreStubUrl"
+              IDENTITY_coreURL: "/perfTest/identity/coreUrl"
+              IDENTITY_STUB_USERNAME: "/perfTest/identity/coreStubUsername"
+              IDENTITY_STUB_PASSWORD: "/perfTest/identity/coreStubPassword"
+              IDENTITY_orchCoreStub: "/perfTest/identity/orchStubUrl"
+              IDENTITY_KBV_URL: "/perfTest/identity/orange/kbvUrl"
+              IDENTITY_ADDRESS_URL: "/perfTest/identity/orange/addressUrl"
+              IDENTITY_ADDRESS_ENV_NAME: "/perfTest/identity/orange/addressEnvName"
+              IDENTITY_KBV_ENV_NAME: "/perfTest/identity/orange/kbvEnvName"
+              IDENTITY_KIWI_URL: "/perfTest/identity/kiwi/kiwiUrl"
               K6_DYNATRACE_APITOKEN: "/perfTest/dynatraceApiToken"
+              K6_DYNATRACE_URL: "/perfTest/dynatraceUrl"
```

Comment:
It would be helpful if all of the environment variables were ordered alphabetically